server/websocket/src/server.rs

```diff
@@ -92,17 +95,38 @@ where
         }));
     }
 
-    fn subgraph_from_url_path(path: PathBuf) -> Option<String> {
-        path.iter()
-            .nth(1)
-            .and_then(|os| os.to_str())
-            .map(|s| s.into())
+    fn subgraph_id_from_url_path(store: Arc<S>, path: PathBuf) -> Result<String, ()> {
```

Comment:
Hmmm true. I think it's OK because the namespaces are separate and similar mostly by coincidence. I could change this to use a matching style similar to server/http though if you like

Reply:
I think it's ok. This logic here is simple compared to that in the GraphQL/HTTP server.